bounties_api/notifications/slack_client.py

```diff
@@ -0,0 +1,89 @@
+from django.conf import settings
+
+from std_bounties.models import Fulfillment, Bounty, BountyState
+from std_bounties.constants import ACTIVE_STAGE
+from notifications.notification_templates import *
+from notifications.constants import *
+from slackclient import SlackClient as _SlackClient
+from .base_clients import BaseClient
+
+
+class Events(Enum):
+    FULFILLMENT_SUBMITTED = 'Bounty Issued'
+    BOUNTY_ACTIVATED = 'Bounty Activated'
+    FULFILLMENT_SUBMITTED = 'Bounty Fulfillment Submitted'
+    BOUNTY_ACTIVATED = 'Bounty Activated'
+    FULFILLMENT_ACCEPTED = 'Bounty Fulfillment Accepted'
+    BOUNTY_EXPIRED = 'Bounty Expired'
+    ISSUE_BOUNTY = 'Bounty Issued'
+    UPDATE_FULFILLMENT = 'Bounty Fulfillment Updated'
+    KILL_BOUNTY = 'Bounty Killed'
+    ADD_CONTRIBUTION = 'Contribution Added'
+    EXTEND_DEADLINE = 'Deadline Extended'
+    CHANGE_BOUNTY = 'Bounty Changed'
+    TRANSFER_ISSUER = 'Issuer Transfered'
+    INCREASE_PAYOUT = 'Payout Increased'
+
+
+class SlackClient(BaseClient):
+    def __init__(self,
+                 client=_SlackClient(settings.SLACK_TOKEN),
+                 channel=settings.NOTIFICATIONS_SLACK_CHANNEL):
+        self.super(SlackClient, self).__init__()
+        self.notify = wrapped_partial(notify_slack,
+                                      client,
+                                      channel)
+
+    def fulfillment_submitted(self, bounty, fulfillment_id):
+        fulfillment = Fulfillment.objects.get(fulfillment_id=fulfillment_id, bounty=bounty)
+        string_data_fulfiller = FULFILLMENT_SUBMITTED_FULFILLER_STR.format(bounty_title=bounty.title)
+        string_data_issuer = FULFILLMENT_SUBMITTED_ISSUER_STR.format(bounty_title=bounty.title)
+
+        self.notify(event=Events.FULFILLMENT_SUBMITTED.value, msg=string_data_issuer)
+
+    def bounty_activated(self, bounty):
```

Comment:
@zoek1 I'm confused - what is this file? Slack notifications should be giving the same message they were giving previously.